vmngclient/session.py

```diff
@@ -107,6 +110,14 @@ def create_vManageSession(
     return session
 
 
+def check_vmanage_server_connection(url):
+    url = f"http://{url}"
```

Comment:
Use `__create_base_url` as its used in every method.

Reply:
Fixed